src/VirtoAppLayout/VirtoAppLayout.tsx

```diff
@@ -101,6 +101,7 @@ function Layout({
         >
           <Box sx={theme => theme.mixins.toolbar} />
           <Box
+            id="page-content"
```

Comment:
I suggest we adopt "component-thing" convention.

```suggestion
            id="virto-app-layout-page-content"
```